_posts/2017-11-28-data-science-assessment-challenges.md

```diff
@@ -0,0 +1,57 @@
+
+More subtly, a data scientist is more than a data analyst. It takes an aptitude for collecting, organizing, and curating data, as well as for thinking analytically. A strong quantitative background is useful but not necessary. Principles and practices from the social sciences or digital humanities are valuable assets; data scientists should be good writers, good storytellers, and good communicators. Perhaps surprisingly, attention to detail is not a key item to include in the data scientist's skillset; ability to grasp the big picture is much more key, as the data scientist will find herself or himself working at the interface of very different departments or fields (in an industry context, these could be engineering, marketing, or business intelligence).
+
+A data scientist does not master any specific technology to perfection, since s/he dabbles in everything! Unlike the traditional data (or business intelligence) analyst, s/he resorts to several different frameworks and programming languages (as opposed to a given domain-specific platform) in order to leverage data. Plus, the data scientist typically works with datasets coming from multiple sources (as opposed to the traditional data analyst who usually works with a single data source already populated by an [ETL](https://en.wikipedia.org/wiki/Extract,_transform,_load) solution). Data scientists are flexible with their tools and approaches.
+
+We had a friendly conversation around SQL. It is ubiquitous in data land, but a data scientist may actually only very rarely write SQL queries (unlike a data engineer, who will do this very often). So, would you want to work in data science, should you learn it? Still, yes! But, do not fret, you can learn it along the way. We can view SQL as our common denominator (in the industries and communities revolving around data). This finding is supported by the above-mentioned report.
+
+In the second part of the meeting, we split into breakout sessions in pairs to discuss the challenges of assessing data science education with respect to the Carpentries workshops. Brainstorming in parallel lets us cover more ground (breadth), while interacting one-on-one lets us explore different avenues (depth).
+
+One pair focused on the industry perspective, another on the education system, and the third on assessment practices. Kari offered a list of questions to frame the discussion. The questions were pulled from the meeting she attended.
```

Comment:
Wondering if we could drop "The questions were pulled from the meeting she attended." since we repeat it line 45 "(pulled from South Big Data Hub's Data Divide workshop)".

Reply:
Agreed!